src/JewellryStore/Main.java

```diff
@@ -0,0 +1,220 @@
+                        System.out.println("(2) Gold");
+                        System.out.println("(3) Platinum");
+                        System.out.println("\n-----------------------------------------");
+                        System.out.print("Enter an option number: ");
+                        int neckMaterial = Integer.parseInt(keyboard.readLine());
+                        if(neckMaterial == 2){
+                            necklaceMaterial = "Gold";
+                        } else if(neckMaterial == 3){
+                            necklaceMaterial = "Platinum";
+                        } else { // If user input is invalid, automatic necklace material is the default material
+                            necklaceMaterial = "Sterling Silver";
+                        }
+                        System.out.println("-----------------------------------------");
+                        System.out.println("\nChoose a chain length: ");
+                        System.out.println("\n(1) 30cm (Short) (Default)");
+                        System.out.println("(2) 40cm (Medium)");
+                        System.out.println("(3) 50cm (Long)");
+                        System.out.println("\n-----------------------------------------");
+                        System.out.print("Enter an option number: ");
+                        int necklaceSize = Integer.parseInt(keyboard.readLine());
+                        if (necklaceSize == 2){ // Size cm calculation for toString
+                            necklaceSize = 40;
+                        } else if (necklaceSize == 3){
+                            necklaceSize = 50;
+                        } else { // If user input is invalid, automatic size is the default size
+                            necklaceSize = 30;
+                        }
+                        System.out.print("\nHow many would you like to purchase?: ");
+                        int necklaceQuantity = Integer.parseInt(keyboard.readLine());
+                        double necklacePrice = (double)necklaceQuantity * 24.99;
+                        if(necklaceSize == 40){  // Adds $2.99 to the price for every 10cm longer chain
+                            necklacePrice += (2.99 * necklaceQuantity);
+                        } else if(necklaceSize == 50){
+                            necklacePrice += (2.99 * 2 * necklaceQuantity);
+                        }
+                        subtotal += necklacePrice;
+                        Necklace necklaceOrder = new Necklace(necklaceSize, necklaceQuantity, necklacePrice, necklaceMaterial);
+                        System.out.println("-----------------------------------------");
+                        System.out.println("\n" + necklaceOrder.toString() + " added to bag.");
+                        totalQuantity += necklaceQuantity;
+                        totalUniqueProducts += 1;
+                        theOrder = new Order(subtotal, totalQuantity);
```

Comment:
why are you creating a new `Order` object every time an item is added to the shopping bag?  The calculation of the total works, however, not why you think it does and not how the Order was designed to work as an object to contain the functionality behind maintaining a list of items to be purchased.